v6_undo_stack.markdown

```diff
@@ -0,0 +1,17 @@
+# Temporary changes for v6
+
+This is a temporary file, containing changes done to release process for v6,
+that need to be reverted back.
+
+It's a file in the repository to allow for easier tracking of the status
+of things among the broader community, and to allow everyone to add/delete
+things they believe should be in here.
+
+The file is scheduled for deletion by the actual release of `v6` and subsequent
+rollback of the changes.
+
+## Stack
+
+* Revert changes to homebrew PR push (`.github/workflows/release-packages.yaml`)
```

Comment:
Commit hashes could be really handy here, assuming that we can avoid invalidating them in a re-base before the PR is merged.

Reply:
I'm not sure if we need commit hashes here - this was supposed to be a casual thing just to verify our reverted changes against the log here, sort of a double-entry bookkeeping practice but for the purpose of managing the process.

But if keen, I can also add this in a new PR.